src/components/page-styles/page-styles.styles.jsx

```diff
@@ -0,0 +1,21 @@
+import styled from 'styled-components';
+
+export const PageTitle = styled.a`
+    font-family: Quicksand;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 36px;
+    line-height: 45px;
+    color: #000000;
+`;
+
+export const Espacamento = styled.div`
+    margin-left: auto;
+    margin-right: auto;
+    margin-top: 35px;
+
+`;
+
+export const Espacamento2 = styled.div`
+    margin-top: 35px;
+`;
```

Comment:
Posição bem ruim, fica ruim para layout responsivos. Utiliza o https://css-tricks.com/snippets/css/a-guide-to-flexbox/

Reply:
hmm não sei muito bem se entendi. a dificuldade na responsividade, pelo que entendi, fica na designação direta do espaçamento (40px, por exemplo). Mas, nesse caso, não é diferente de um Breakline, já que a designação de espaços está sendo feita para o sentido vertical... (inclusive, 35px é menor que um breakline, que possui espaço de 48px por linha.